### ISSUE TITLE
build failure with ip-1.0.0

### ISSUE THREAD

**Issue by DanBurton** [athanclark/attoparsec-uri#1]
Build failures as seen on the stackage build server:

```
Building library for attoparsec-uri-0.0.2..
[1 of 3] Compiling Data.URI.Auth.Host ( src/Data/URI/Auth/Host.hs, dist/build/Data/URI/Auth/Host.o )

src/Data/URI/Auth/Host.hs:21:1: error:
    Could not find module ‘Net.IPv4.Text’
    Use -v to see a list of the files searched for.
   |
21 | import qualified Net.IPv4.Text as IPv4
   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

src/Data/URI/Auth/Host.hs:22:1: error:
    Could not find module ‘Net.IPv6.Text’
    Use -v to see a list of the files searched for.
   |
22 | import qualified Net.IPv6.Text as IPv6
   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

**Comment by athanclark** [comment]
This should be fixed now - attoparsec-uri >= 0.0.3 now depends on attoparsec-ip >= 0.0.1, which depends on ip >= 1.1.0

**Comment by juhp** [comment]
For 1.1.0 the benchmarks fail to build:

```
test/IPv4TextVariableBuilder.hs:10:1: error:
    Could not find module ‘Data.Text.Builder.Variable’
    it is a hidden module in the package ‘ip-1.1.0’
    Use -v to see a list of the files searched for.
   |
10 | import qualified Data.Text.Builder.Variable as VB
   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```
